core/src/test/java/org/apache/accumulo/core/metadata/schema/TabletMetadataTest.java

```diff
@@ -116,7 +116,7 @@ public void testAllColumns() {
     SortedMap<Key,Value> rowMap = toRowMap(mutation);
 
     TabletMetadata tm = TabletMetadata.convertRow(rowMap.entrySet().iterator(),
-        EnumSet.allOf(ColumnType.class), true);
+        EnumSet.allOf(ColumnType.class), true, false);
```

Comment:
If not covered elsewhere, a unit test for the error suppression case would be nice.

Reply:
Test added in 9153726